pkg/grpc/device_definition.proto

```diff
@@ -151,6 +151,8 @@ message Integration {
     int32 ICE = 3;
     int32 PHEV = 4;
   }
+  int64 points = 9;
+  uint64 manufacturer_token_id = 10;
```

Comment:
don't know if we care about this being a uint64 here but big.Int elsewhere. my guess is it'll be fine b/c we won't ever have millions of different manufacturers. 

https://www.reddit.com/r/golang/comments/awtxw8/whats_the_best_way_to_add_a_bigint_to_a_protobuf/

Reply:
Oh there are Protobuf extensions:

https://github.com/nutsfinance/solidity-protobuf